File: app/src/main/java/se/creotec/chscardbalance2/model/OnCardDataChangedListener.java
// Copyright (c) 2017 Alexander Håkansson
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT
package se.creotec.chscardbalance2.model;

public interface OnCardDataChangedListener {
    void notify(final CardData newData);
}

File: app/src/main/java/se/creotec/chscardbalance2/model/OnCardDataChangedListener.kt
// Copyright (c) 2017 Alexander Håkansson
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT
package se.creotec.chscardbalance2.model

interface OnCardDataChangedListener {
    fun cardDataChanged(newData: CardData)
}
